packages/sitecore-jss/src/feaas/themes.test.ts

```diff
@@ -63,370 +68,416 @@ describe('utils', () => {
+                CSSStyles: '-library--foo',
+              },
+            })
+          )
+        ).to.deep.equal([{ href: getStylesheetUrl('foo'), rel: 'style' }]);
+      });
 
-        it('should return links using LibraryId param', () => {
-          expect(
-            getFEAASLibraryStylesheetLinks(
-              setBasicLayoutData({
-                componentName: 'styled',
-                params: {
-                  LibraryId: 'bar',
-                },
-              })
-            )
-          ).to.deep.equal([{ href: getStylesheetUrl('bar'), rel: 'style' }]);
-        });
+      it('should return links using LibraryId param', () => {
+        expect(
+          getFEAASLibraryStylesheetLinks(
+            setBasicLayoutData({
+              componentName: 'styled',
+              params: {
+                LibraryId: 'bar',
+              },
+            })
+          )
+        ).to.deep.equal([{ href: getStylesheetUrl('bar'), rel: 'style' }]);
+      });
 
-        it('should return prefer params over fields', () => {
-          expect(
-            getFEAASLibraryStylesheetLinks(
-              setBasicLayoutData({
-                componentName: 'styled',
-                params: {
-                  CSSStyles: '-library--foo',
-                },
-                fields: {
-                  CSSStyles: {
-                    value: '-library--not-foo',
-                  },
+      it('should return prefer params over fields', () => {
+        expect(
+          getFEAASLibraryStylesheetLinks(
+            setBasicLayoutData({
+              componentName: 'styled',
+              params: {
+                CSSStyles: '-library--foo',
+              },
+              fields: {
+                CSSStyles: {
+                  value: '-library--not-foo',
                 },
-              })
-            )
-          ).to.deep.equal([{ href: getStylesheetUrl('foo'), rel: 'style' }]);
+              },
+            })
+          )
+        ).to.deep.equal([{ href: getStylesheetUrl('foo'), rel: 'style' }]);
 
-          expect(
-            getFEAASLibraryStylesheetLinks(
-              setBasicLayoutData({
-                componentName: 'styled',
-                params: {
-                  LibraryId: 'bar',
-                },
-                fields: {
-                  LibraryId: {
-                    value: 'not-bar',
-                  },
+        expect(
+          getFEAASLibraryStylesheetLinks(
+            setBasicLayoutData({
+              componentName: 'styled',
+              params: {
+                LibraryId: 'bar',
+              },
+              fields: {
+                LibraryId: {
+                  value: 'not-bar',
                 },
-              })
-            )
-          ).to.deep.equal([{ href: getStylesheetUrl('bar'), rel: 'style' }]);
-        });
+              },
+            })
+          )
+        ).to.deep.equal([{ href: getStylesheetUrl('bar'), rel: 'style' }]);
```

Comment:
If we are testing the results of getStylesheetUrl, does it make sense to call it directly like this in unit tests?
I.e. if there's smth wrong with getStyleheetUrl implementation this test will not throw an error

Reply:
@art-alexeyenko since this unit test uses _getFEAASLibraryStylesheetLinks_, so if _getStylesheetUrl_ is broken, it will be broken in any case, even if we will hardcode "href" here. Right? Dependency is still here

Reply:
or may be I missed your point

Reply:
yep, right - and we test getStylesheetUrl separately so we should be good.